web/tailwind.config.ts

```diff
@@ -194,27 +193,29 @@ export default defineConfig({
           200: 'var(--wp-link-200)',
         },
       },
+      fontFamily: {
+        sans: [
+          'system-ui',
+          '-apple-system',
+          'Segoe UI',
+          'Roboto',
+          'Helvetica Neue',
+          'Noto Sans',
+          'Liberation Sans',
+          'Arial',
+          'sans-serif',
+        ],
+      },
+      borderWidth: {
+        6: '6px',
+      },
```

Comment:
This seems to be new. Where is it coming from?

Reply:
Its a custom class required to make some existing custom windi classes work.

Reply:
Can you please add more details? Where do you got this from?

Reply:
There is no tailwind class with a border width of 6px.

Reply:
Sorry I still dont get it. Where exactly do we need a border with the width of 6px and how is it used?

Reply:
Then is suggest to use 4 or 8 if thats the only location.

Reply:
There's another one, best to grep for it. Maybe something for a different PR?